shared/cfg/guest-os/Windows/Win10/i386.cfg

```diff
@@ -37,3 +36,5 @@
         uninstall_balloon_service = "%s:\Balloon\w10\x86\blnsvr.exe -u"
         status_balloon_service = "%s:\Balloon\w10\x86\blnsvr.exe status"
         run_balloon_service = "%s:\Balloon\w10\x86\blnsvr.exe -r"
+    balloon_check, balloon_stop_continue, balloon_stress, balloon_hotplug, balloon_in_use, balloon_service, balloon_illegal, balloon_boot_in_pause, systemtap_tracing.balloon_event, driver_load_stress.with_balloon, win_virtio_driver_update_test.with_balloon:
+        vm_mem_limit = 3G
```

Comment:
1. It will not affect other test cases, only balloon related cases.
2. confirmed with the developer and owner, only update balloon test part.

Thanks.